edx_adapt/api/resources/course_resources.py

```diff
@@ -170,7 +170,7 @@ def get(self, course_id):
         try:
             exps = self.repo.get_experiments(course_id)
         except DataException as e:
-            print("--------------------\tDATA EXCEPTION: " + str(e))
+            logger.error("Data exception: {}".format(e))
```

Comment:
You can use `logger.exception()` here. It will add exception details for automatically. You just need to make sure to write helpful message.